spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -113,4 +113,21 @@
     expect(valid_response).to match_response_schema("collection")
     expect(invalid_response).not_to match_response_schema("collection")
   end
+
+  it "works when a pure string is used for response" do
+    create_schema("foo_schema", {
+      "type" => "object",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.